app/models/person.rb

```diff
@@ -92,4 +94,33 @@ def picture_size
 
     errors.add(:picture, 'grösse kann maximal 10MB sein')
   end
+
+  class << self
+    def from_omniauth(auth) # rubocop:disable Metrics/AbcSize
+      person = where(email: auth.info.email).first_or_create do |user|
+        user.name = auth.info.name
+        user.birthdate = DateTime.new(2020, 1, 1)
```

Comment:
We have to make many assumptions here which isn't nice.
Do we need to create a Person for the login to make sense?